packages/ckeditor5-bookmark/src/bookmarkediting.ts

```diff
@@ -92,6 +96,8 @@ export default class BookmarkEditing extends Plugin {
 		const { editor } = this;
 		const { conversion, t } = editor;
 
+		editor.data.upcastDispatcher.on<UpcastElementEvent>( 'element:a', dataViewModelAnchorInsertion( editor ) );
```

Comment:
I'd suggest using the official public API for conversion:
```suggestion
		conversion.for( 'upcast' ).add( dispatcher => dispatcher.on<UpcastElementEvent>( 'element:a', dataViewModelAnchorInsertion( editor ) );
```

Reply:
I've moved it also into right place.